.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "Python 3",
-  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-  "image": "mcr.microsoft.com/devcontainers/python:1-3.11-bullseye",
+  "image": "python:3.11-bullseye",
```

Comment:
I updated this locally because I am using cursor, not VSCode. Apparently, because cursor isn't a microsoft product, I wasn't able to access the container from them. Happy to roll this back if desired.